examples/v2-minimal/packages/contracts/mud.config.mts

```diff
@@ -15,6 +15,13 @@ export default mudConfig({
       },
       storeArgument: true,
     },
+    EventTable: {
+      name: "event",
+      schema: {
+        value: "bool",
+      },
+      ephemeral: true,
```

Comment:
Still not entirely sold on "ephemeral" as the name for this pattern but happy to go with this for now (haven't come up with anything better)

Reply:
i've been thinking about it and also haven't come up with anything

Reply:
ask chatgpt

Reply:
I would have proposed "transient" table, but I feel like we'll need this name for tables that use Solidity's upcoming transient storage